docs/plugins.yml

```diff
@@ -0,0 +1,78 @@
+---
+#####################
+# We're excited you wrote plugin! Here's how you share it with others:
+#
+# 1. Add a newline anywhere in the list below
+# 2. List your plugin following the <user>/<repo> format
```

Comment:
>Currently the order matters. If we just sort alphabetically, ircody/... would be higher on the list than intelsdi-x/... So would it be better to follow the order as it appears here so you have some control?

How do we differentiate between process/publisher then? It looks at the name of the plugin right? It also seems like it'd be pretty easy to sort on what comes after the `/`?